### PR TITLE
Fix guides

### PR DESCRIPTION
**Pull request by timetherewere**
The guides are ready to be decimated by @woodser and @trasherdk.

### CODE REVIEW

File: docs/developer_guide/getting_started.md [hunk 1]
```diff
@@ -2,19 +2,15 @@
...
+![monero-javascript hierarchy](../architecture.png?raw=true)*monero-javascript uses Monero through three channels: RPC wallet servers, RPC daemon servers, and Wasm wallets.*  
 
 ## Initial Setup
 
 ### Install Node.js and the Node package manager (npm)
 
-Make sure you have Node.js and npm installed; you need them to obtain and use the monero-javascript library. See the ["Node.js and npm"](https://github.com/monero-ecosystem/monero-javascript/blob/master/docs/developer_guide/installing_prerequisite_software.md#nodejs-and-npm) section of the prerequisite installation guide for step-by-step instructions for downloading and installing Node.js and npm.
+Make sure you have Node.js and npm installed; you need to install Node.js and npmbefore you can install and use the monero-javascript library. See the ["Node.js and npm"](https://github.com/monero-ecosystem/monero-javascript/blob/master/docs/developer_guide/installing_prerequisite_software.md#nodejs-and-npm) section of the prerequisite installation guide for step-by-step instructions for downloading and installing Node.js and npm.
```
Comment (trasherdk) [inline docs/developer_guide/getting_started.md]: `Node.js and npmbefore` => `Node.js and npm before`

File: docs/developer_guide/getting_started.md [hunk 2]
```diff
@@ -58,24 +56,24 @@ The asynchronous function is not strictly necessary, but most applications need
 
 ### Building a keys-only wallet
 
-Monero-javscript implements keys-only wallets in the MoneroWalletKeys class. You can create a random keys-only wallet by calling the [MoneroWalletKeys](moneroecosystem.org/monero-javascript/MoneroWalletKeys.html) class's `createWallet()` method in mainFunction() as follows:
+Monero-javscript implements keys-only wallets in the MoneroWalletKeys class. You can create a random keys-only wallet by calling the [MoneroWalletKeys](https://moneroecosystem.org/monero-javascript/MoneroWalletKeys.html) class's `createWallet()` method in mainFunction() as follows:
```
Comment (woodser) [inline docs/developer_guide/getting_started.md:59]: Uppercase M inconsistent with other instances.

File: docs/developer_guide/getting_started.md [hunk 3]
```diff
@@ -58,24 +56,24 @@ The asynchronous function is not strictly necessary, but most applications need
...
-Use the Wallet class's getter methods to obtain and log the wallet's basic attributes. These attributes are:
-* The seed phrase
+The offline wallet generator should display four basic wallet attritubes:
+* The mnemonic seed phrase
 * The address
 * The spend key
 * The view key
 
-Get each wallet attribute and log it in the console.
+Use the Wallet class's getter methods to obtain and log the wallet's basic attributes, and log them in the console.
```
Comment (woodser) [inline docs/developer_guide/getting_started.md:73]: "Wallet" should not be capitalized.

File: docs/developer_guide/getting_started.md [hunk 4]
```diff
@@ -89,26 +87,26 @@ await mainFunction();
 
 async function mainFunction() {
   // create a random keys-only (offline) stagenet wallet
-  var walletKeys = await MoneroWalletKeys.createWallet({networkType: MoneroNetworkType.STAGENET, language: "English"});
+  let walletKeys = await MoneroWalletKeys.createWallet({networkType: MoneroNetworkType.STAGENET, language: "English"});
 
-  console.log("Seed phrase: " + await(walletKeys.getMnemonic()));
-  console.log("Address: " + await(walletKeys.getAddress(0,0))); // MoneroWallet.getAddress(accountIndex, subAddress)
+  console.log("Mnemonic seed phrase: " + await(walletKeys.getMnemonic()));
```
Comment (woodser) [inline docs/developer_guide/getting_started.md:92]: `await(walletKeys.getMnemonic())` should be `await walletKeys.getMnemonic()`, same for other instances.

File: docs/developer_guide/installing_prerequisite_software.md [hunk 5]
```diff
@@ -1,62 +1,41 @@
...
+### For Fedora:
+
   `$ sudo dnf install python3`
 
 ## Node.js and npm
-_explanation of necessity here_
+
+JavaScript is designed to run client-side in web browsers, so JavaScript programs cannot run as standalone or server-side applications out of the box. Node.js solves this limitation by providing a runtime environment, allowing JavaScript programs to execute outside of a web browser.
+
+[Part 1 of the monero-JavaScript "getting started" guide]() illustrates the use of Node.js to run a monero-JavaScript program in a unix terminal, so you need to install it to follow the guide.
```
Comment (woodser) [inline docs/developer_guide/installing_prerequisite_software.md:22]: Empty link.

File: docs/developer_guide/web_app_guide.md [hunk 6]
```diff
@@ -1,66 +1,108 @@
-# Getting Started with Monero Javascript part 2: Creating a Web Application
+# Getting Started with Monero Javascript Part 2: Creating a Web Application
```
Comment (woodser) [inline docs/developer_guide/web_app_guide.md:1]: The titles of other guide documents use lowercase after the first word.  Shall we change this or those to be consistent?

File: docs/developer_guide/web_app_guide.md [hunk 7]
```diff
@@ -1,66 +1,108 @@
...
+Alternately, you can [manually download the script.](https://github.com/timetherewere/monero-javascript/raw/include_compressed_template/web_template_script)
 
-The script will configure the project folder and serve the starter web application on port 9100. Open a web browser and type "localhost:9100" in the URL bar to view the application.
+The script configures a project folder and serves two example web applications on port 9100. Open a web browser and navigate to "localhost:9100" for links to the applications:
+* "Sample code" demonstrates a handful of common monero-javascript operations. Open the developer console to see the application's output.
+* "Offline wallet generator" shows off the final result of following this guide. If you wish look over the offline wallet generator source code located in "src/offline_wallet_generator.html" and "src/offline_wallet_generator.js" and skip to the [Additional Resources](#additional_resources) section of this guide. Otherwise, ignore the wallet generator app and follow the rest of the guide to build it yourself from scratch.
```
Comment (woodser) [inline docs/developer_guide/web_app_guide.md:36]: "If you wish, look over..."

File: docs/developer_guide/web_app_guide.md [hunk 8]
```diff
@@ -1,66 +1,108 @@
...
+
+```
+require("monero-javascript");
+
+await mainFunction();
 
-`<script type="text/javascript" src="xmr-sample-app.js"></script>`
+async function mainFunction() {
+  // create a random keys-only (offline) stagenet wallet
+  var walletKeys = await MoneroWalletKeys.createWallet({networkType: MoneroNetworkType.STAGENET, language: "English"});
```
Comment (woodser) [inline docs/developer_guide/web_app_guide.md:79]: `let` instead of `var`

File: docs/developer_guide/getting_started.md [hunk 9]
```diff
@@ -32,9 +28,11 @@ Make sure you have Node.js and npm installed; you need them to obtain and use th
 
 ### The offline wallet generator
 
-An offline wallet generator creates and displays a new, random wallet address along with that address's associated view key, spend key, and mnemonic seed phrase. An offline wallet generator only needs to generate and display these wallet attributes; it does not need to communicate with a Monero network, transfer XMR, or track a wallet's balance or outputs.
+An offline wallet generator creates and displays a new, random wallet address along with that address's associated view key, spend key, and mnemonic seed phrase. An offline wallet generator only needs to produce and display these wallet attributes; it does not need to communicate with a Monero network, transfer XMR, or track a wallet's balance or outputs.
+
+monero-javascript's keys-only wallet more than meets these requirements and is the ideal basis for a monero-javascript-based offline wallet generator. The keys-only wallet is a minimal Wasm wallet implementation that only tracks a wallet's _permanent_ attributes. It cannot initiate transfers, report its balance, or communicate with a Monero network.
```
Comment (woodser) [inline docs/developer_guide/getting_started.md:33]: "...keys-only wallet meets these requirements..." ?

File: docs/developer_guide/getting_started.md (hunk 1 above)
Comment (timetherewere) [reply docs/developer_guide/getting_started.md]: What the... this was one of the things I fixed! Darnit...

Comment (timetherewere) [reply docs/developer_guide/getting_started.md]: Ah, perhaps that's why it says "outdated".

File: docs/developer_guide/web_app_guide.md (hunk 6 above)
Comment (timetherewere) [reply docs/developer_guide/web_app_guide.md:1]: Changing to lower. The Monero user guides and dev docs for other platforms use lower.

**Pull request merged**